src/modules/dpos/validator/api/index.test.js

```diff
@@ -101,7 +109,9 @@ describe('API: LSK Delegates', () => {
       await delegate.getDelegates({
         network,
         params: {
-          addressList, limit: 5, offset: 3,
+          addressList,
+          limit: 5,
+          offset: 3,
```

Comment:
As explained above, reformatting a code that you do not intend to refactor is a bad habit. You should refrain. But Additionally, I don't get how your formater is configured. In some places, you have put objects with up to 3 members in a single line, and in some cases you have broken  them down into multiple lines. This is inconsistent. 
I'd recommend you revert all of them.

Reply:
okay, would revert. 
Though this happens because my prettier formatter which uses the prettier config on the project formats on save.